cms/envs/bok_choy.py

```diff
@@ -77,6 +77,15 @@
 YOUTUBE['TEST_URL'] = "127.0.0.1:{0}/test_youtube/".format(YOUTUBE_PORT)
 YOUTUBE['TEXT_API']['url'] = "127.0.0.1:{0}/test_transcripts_youtube/".format(YOUTUBE_PORT)
 
+FEATURES['ENABLE_COURSEWARE_INDEX'] = True
+SEARCH_ENGINE = "search.tests.mock_search_engine.MockSearchEngine"
+# Path at which to store the mock index - can remove disable=invalid-name
+# once global settings are set to disable this error and
+# disable=no-value-for-parameter follow the pattern above
```

Comment:
Do you mean in common.py? I felt that it would be better to have it set to None by default, so that indexing operations do not get in the way of other tests. In particular, we found that when we were actively backing the mock index by file that the file was growing very large, which was slowing down all tests significantly. It is possible that I was too aggressive backing it out of everything else, and we did make the adjustment to only do the file-backed index when a test required it, but it did seem safest to test search only when we desired it.